pkg/cloudprovider/cloudprovider.go

```diff
@@ -345,8 +345,8 @@ func (c *CloudProvider) instanceToNodeClaim(i *instance.Instance, instanceType *
 	if v, ok := i.Tags[corev1beta1.NodePoolLabelKey]; ok {
 		labels[corev1beta1.NodePoolLabelKey] = v
 	}
-	if v, ok := i.Tags[corev1beta1.ManagedByAnnotationKey]; ok {
-		annotations[corev1beta1.ManagedByAnnotationKey] = v
+	if v, ok := i.Tags[v1beta1.EKSClusterNameAnnotationKey]; ok {
+		annotations[v1beta1.EKSClusterNameAnnotationKey] = v
```

Comment:
Confirming: We can use a key value with a colon? IIRC, you mentioned that this wasn't possible and would cause failures?

Reply:
We cannot have a key value with a colon. After looking more, I don't think this annotation is actually used anywhere, wdyt about remove it entirely?